lib/jormungandr/test/unit/Cardano/Wallet/Jormungandr/TransactionSpec.hs

```diff
@@ -478,6 +506,27 @@ goldenTestStdTx tl keystore inps outs bytes' = it title $ do
   where
     title = "golden test mkStdTx: " <> show inps <> show outs
 
+goldenTestDelegationCertTx
+    :: forall t k. (t ~ Jormungandr, HasCallStack)
+    => TransactionLayer t k
+    -> (Address -> Maybe (k 'AddressK XPrv, Passphrase "encryption"))
+    -> PoolId
+    -> (k 'AddressK XPrv, Passphrase "encryption")
+    -> [(TxIn, TxOut)]
+    -> [TxOut]
+    -> ByteString
+    -> SpecWith ()
+goldenTestDelegationCertTx tl keystore pool (accountXPrv, pass) inps outs bytes' = it title $ do
+    let res = mkDelegationCertTx tl pool (accountXPrv, pass) keystore inps outs
+    let sealed = getSealedTx . snd <$> res
+    sealed `shouldBe` (Right $ unsafeFromHex bytes')
```

Comment:
**[suggestion]**

Compare hex-encoded bytestrings instead of decoded bytestrings

**Why?**

This makes the counter-example output automatically more readable via `shouldBe` (for what it's worth). Also, this is more consistent with the other golden test:

```hs
goldenTestStdTx tl keystore inps outs bytes' = it title $ do
    let tx = mkStdTx tl keystore inps outs
    let bytes = hex . getSealedTx . snd <$> tx
    bytes `shouldBe` Right bytes'
```

Reply:
From spending much time looking at these failures, I found the hex-encoded version unusable because it doesn't highlight where the changes are within the strings.

Looking at the raw byte string gives me a quick indication of how much/where the differences are in the binary. If I needed to have a closer look, I'd write them to files and compare with [HexFiend](http://ridiculousfish.com/hexfiend/).

It feels like we should be able to have the same highlighting when hex-encoded, but I'm unable to find any trace on google.

<img width="1440" alt="Skärmavbild 2019-11-26 kl  12 02 32" src="https://user-images.githubusercontent.com/304423/69624598-4551d480-1045-11ea-9d5f-8a534a3f578e.png">
<img width="1440" alt="Skärmavbild 2019-11-26 kl  12 06 57" src="https://user-images.githubusercontent.com/304423/69624600-471b9800-1045-11ea-9e50-062e5ef129aa.png">

Reply:
Indeed. That's weird. I've tried using some alternatives by comparing list instead (e.g. `BS.unpack bytes' shouldBe BS.unpack bytes`) but the output is kinda unsatisfactory.